### ISSUE TITLE
Static Form Compatibility Issues with Sulu 2.0

### ISSUE THREAD

**Issue by marcostastny** [sulu/SuluFormBundle#233]
| Q | A
| --- | ---
| Bug? | yes
| New Feature? | no
| Bundle Version | develop @ 3ec157a943002242f76f606888ce72b3c1732a8f
| Sulu Version | 2.0.4
| Browser Version | Browser independent

#### Actual Behavior

When trying to display a static form under the new Sulu 2.0 (for example after Upgrading) the form will break with the following Error:
```
The "Sulu\Bundle\FormBundle\Controller\FormWebsiteController::getParameter()" method is missing a parameter bag to work properly. Did you forget to register your controller as a service subscriber? This can be fixed either by using autoconfiguration or by manually wiring a "parameter_bag" in the service locator passed to the controller.
```

#### Expected Behavior

Either the configuration should be fixed, which is currently breaking the static forms, or it would be nice if the Documentation gets adapted with how to quickly change a static form to a Symfony Form.

#### Steps to Reproduce

I configured the bundle as follows:

In contact-form.xml (page template)
```
<controller>Sulu\Bundle\FormBundle\Controller\FormWebsiteController:formAction</controller>
```

In services.yml
```
contact_form:
        class: 'App\Form\Type\ContactRequestFormType'
        arguments:
            - '@translator'
        tags:
            - 'form.type'
```

Configure a Form Type and Entity like in the Documentation.

#### Possible Solutions

A possible solution would be to update the Documentation to also contain a section Upgrade Static forms to Symfony forms.

To fix static forms you will probably need to inject the `parameter_bag` service in `vendor/sulu/sulu-form-bundle/DependencyInjection/SuluFormExtension.php`.

**Comment by luca-rath** [comment]
@marcostastny can you verify, if #235 is working for you?

**Comment by alexander-schranz** [comment]
@marcostastny should be fixed with #235

**Comment by marcostastny** [comment]
@alexander-schranz @luca-rath Thank you for looking into it!
